src/client/public/styles/nav.css

```diff
@@ -1,11 +1,16 @@
 nav-element {
-  width: 250px;
+  width: var(--nav-size);
   display: flex;
   flex-direction: column;
   margin-right: var(--grid-size);
   font-size: 14px;
   line-height: 20px;
   flex-shrink: 0;
+  position: absolute;
+
+  --grid-size: 16px;
+  --max-desktop-width: 1024px;
+  --nav-size: 250px;
```

Comment:
Stupid question - will these variables be picked up even though they are defined *after* usage?

Reply:
Removed, shouldn't be there.